episodes/1-introduction.Rmd

```diff
@@ -57,15 +57,17 @@ A neural network consists of connected computational units called **neurons**. E
 - most of the time, each neuron conducts 3 main operations:
   + take the weighted sum of the inputs where ($w_1, w_2, ... $) indicate weights
   + add an extra constant weight (i.e. a bias term) to this weighted sum
-  + apply a non-linear function to the output so far (using a predefined activation function such as the ReLU function)
+  + apply a non-linear function to the output so far (using a predefined activation function such as the [ReLU](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)) function)
```

Comment:
This will be changed again with #498 , but approving for now.

Reply:
Indeed, the link is unnecessary now. Merged.